components/coachindicator/stories/template.js

```diff
@@ -0,0 +1,27 @@
+import { html } from "lit-html";
+import { classMap } from "lit-html/directives/class-map.js";
+
+import "../index.css";
+
+export const Template = ({
+	rootClass = "spectrum-CoachMark",
```

Comment:
Rather than adding "Indicator" onto this in the `classMap`, would it be more clear to just set the `rootClass` to `spectrum-CoachMarkIndicator`? Here and also in the `coachindicator.stories.js` args.